src/views/App/styles.scss

```diff
@@ -4,18 +4,18 @@
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-}
-
 .App-header {
```

Comment:
Just note that once @Kevinjhlin navbar branch is merged, then this class won't matter anymore